lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -463,6 +463,15 @@ def add_creditcard(xml, creditcard, currency=nil)
         end
       end
 
+      def add_cdpt_authentication_value(xml, creditcard)
+        xml.tag! :AuthenticationECIInd, nil
```

Comment:
> the docs say "the ECI Indicator field can be left empty and the appropriate value is derived by the gateway."

Interesting. Gateways seem all over the place when it comes to accepting `eci`. I'd still vote for including it since it's actually a standard field on a `NetworkTokenizationCreditCard` which still leaves the option for the value to be `nil` depending on how the `NetworkTokenizationCreditCard` was initialized.